.buildkite/pipeline.yml

```diff
@@ -14,9 +14,12 @@ steps:
     key: "bundle-android"
     plugins:
       - docker#v3.8.0:
-          image: "public.ecr.aws/automattic/gb-mobile-image:latest"
+          image: "public.ecr.aws/automattic/gb-mobile-image:6804aad7632be7db25870cdf283ed310f0c8380e"
```

Comment:
Once https://github.com/wordpress-mobile/docker-gb-mobile-image/pull/4 is merged and a new Docker image version is released, we should revert this change.

Reply:
The PR was merged so the version has been changed to `latest` in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4411/commits/40fa7390b95a6d6db2b148ccadff7abc6e62ad9e.